### ISSUE TITLE
No possibility or intuitive way to rate the course module

### ISSUE THREAD

**Issue by adamvm** [coders-school/site#84]
Course interface includes some stars for rating modules

![image](https://user-images.githubusercontent.com/5563693/84559152-ae2a4100-ad38-11ea-8eda-f941252bb3aa.png)

but there is no way or it is not intuitive how to change state of 
![image](https://user-images.githubusercontent.com/5563693/84559173-ce5a0000-ad38-11ea-9c42-2ab85e34f2a5.png)

**Comment by ziobron** [comment]
That's true, but even I don't know how to enable course ratings :/ We may just remove these stars until we get to know that :D
